portfolio/src/main/webapp/script.js

```diff
@@ -48,7 +48,7 @@ function handleResponse(response) {
 
 /** Adds the greeting to the DOM. */
 function addGreetingToDom(greeting) {
-  console.log('Adding greeting: '+greeting);
+  console.log('Adding greeting: '+ greeting);
```

Comment:
nit: one more space before the `+` please :)

Reply:
Oh great spot, I'll add it now!